framework-docs/modules/ROOT/pages/web/websocket/stomp/application-context-events.adoc

```diff
@@ -8,7 +8,7 @@ received by implementing Spring's `ApplicationListener` interface:
 While the "`simple`" broker becomes available immediately on startup and remains so while
 the application is running, the STOMP "`broker relay`" can lose its connection
 to the full featured broker (for example, if the broker is restarted). The broker relay
-has reconnect logic and re-establishes the "`system`" connection to the broker
+has reconnected logic and re-establishes the "`system`" connection to the broker
```

Comment:
"reconnect" was actually correct: it has logic that knows how to _reconnect_.

I suppose you could call it "reconnect-logic" (hyphenated) or leave out the part about "reconnect logic" since the next part discusses re-establishing the connection anyway.